fastlane/swift/Gymfile.swift

```diff
@@ -18,4 +18,4 @@ class Gymfile: GymfileProtocol {
 
 
 
-// Generated with fastlane 2.85.0
```

Comment:
Weird that it is stripping out the version entirely, let's see what the `version bump` pr does with it. Might be a bug somewhere. I think I've seen this before